sites/shared/components/workbench/draft/defs.mjs

```diff
@@ -2,28 +2,47 @@ const style = ` style="fill: none; stroke: currentColor;" `
 const grids = {
   imperial: `
       <pattern id="grid" height="25.4" width="25.4" patternUnits="userSpaceOnUse" key="grid">
-        <path class="gridline lg imperial" d="M 0 0 L 0 25.4 L 25.4 25.4" ${style} />
+        <!-- lg: solid lines every 1 inch -->
+        <path class="gridline lg imperial"
+          d="M 0 0 L 0 25.4 L 25.4 25.4"
+          ${style}
+        />
+        <!-- md: dashed lines every 1/2 inch -->
         <path
-          class="gridline lg imperial"
+          class="gridline md imperial"
```

Comment:
There's should be no `md` class. The absence of a size modifier (so just `gridline`) is what makes is the middle size